docs/manual/java/guide/cluster/index.toc

```diff
@@ -1,4 +1,5 @@
 PersistentEntity:Persistent Entity
+ChoosingADatabase:Choosing A Database
```

Comment:
Should this be above `PersistentEntity` since it was one of the first sections there? Or maybe just have a link at `PersistentEntity.md` page.

Reply:
I didn't pay too much attention to ordering as we'll be shaking all this content a bit in the next few days.

I'll make the PR self-contained. :-)

Reply:
I'm actually tempted to move the `ChoosingADatabase` all the way down next to `Cluster` or `Serialization`.

The more I look at it the more I think this whole section on clustered services needs a deeper reordering. I think we're better of merging the RP as is.

Reply:
This is the current look of `Cluster Services` section in the scala version of the docs:

![Screen Shot 2019-10-30 at 11 18 16](https://user-images.githubusercontent.com/762126/67849539-047aa480-fb07-11e9-93f5-fdcdb0f7f900.png)